tests/formats/kzg/compute_kzg_proof.md

```diff
@@ -10,14 +10,15 @@ The test data is declared in a `data.yaml` file:
 input:
   blob: Blob -- the data blob representing a polynomial
   z: Bytes32 -- bytes encoding the BLS field element at which the polynomial should be evaluated
-output: KZGProof -- The KZG proof
+output: Tuple[KZGProof, y] -- The KZG proof and the value y = f(z)
```

Comment:
How do you feel about splitting these up like this:

```suggestion
output:
  proof: KZGProof -- The KZG proof
  y: Bytes32 -- The value y = f(z)
```

Regardless, you will want to change `y` to `Bytes32`.

Reply:
The reason I did not write it like this is that return types typically don't have names -- it's just a tuple of two values. I don't have a strong opinion either way, though.

And yeah it should be `Tuple[KZGProof, Bytes32]`.

Reply:
That's fair 👍 makes sense to me.